ssl/statem/statem_clnt.c

```diff
@@ -271,13 +384,94 @@ int ossl_statem_client_read_transition(SSL *s, int mt)
 }
 
 /*
- * client_write_transition() works out what handshake state to move to next
- * when the client is writing messages to be sent to the server.
+ * ossl_statem_client13_write_transition() works out what handshake state to
+ * move to next when the TLSv1.3 client is writing messages to be sent to the
+ * server.
+ *
+ * Return values:
```

Comment:
So the return value is set by the WRITE_TRAN enum, which should document the error, continue, finished choices.  I prefer us not to duplicate text. :)